deployment/GS01SUMO_Deploy.csv

```diff
@@ -160,6 +160,7 @@ DY096,,DY112,,GS01SUMO-RID16-05-PCO2WB000,4,1,2018-12-04T17:10:40,2020-01-20T10:
 DY096,,DY112,,GS01SUMO-RID16-06-DOSTAD000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-DOSTAD-00507,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID16-07-NUTNRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-NUTNRB-01103,-54.407168,-89.206037,,12,4589,
 DY096,,DY112,,GS01SUMO-RID16-08-SPKIRB000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGINS-SPKIRB-00291,-54.407168,-89.206037,,12,4589,
+DY096,,DY112,,GS01SUMO-RIC21-00-CPMENG000,4,1,2018-12-04T17:10:40,2020-01-20T10:01:00,CGMGS-01SUMO-00004,,CGCON-NCPM02-00013,-54.407168,-89.206037,,12,4589,

```

Comment:
To be consistent with other stuff added for NOC, add Note:
"Extra CPM for NOC deployment"

Reply:
comment added!